NetKAN/KSP-Slovakia-flags

```diff
@@ -0,0 +1,21 @@
+{
+    "spec_version" : "v1.2",
+    "identifier"   : "KSP Slovakia flags",
+    "$kref"        : "#/ckan/github/miki-g/KSP-Slovakia-Flags",
+    "$vref"        : "#/ckan/ksp-avc",
```

Comment:
Can you replace this line with the following?

    "ksp_version"    : "any",
    "comment"        : "flagmod",